api/src/services/healthService.js

```diff
@@ -0,0 +1,83 @@
+const { blockchainNames, networkNames, accountGroups } = require('../config/blockchains');
+const accountsService = require('./accountsService');
+
+async function checkAccountsBalances(req, res) {  
+  try {
+    const { query } = req;
+    const blockchains = query.blockchains && query.blockchains.split('|');
```

Comment:
Use `url.parse` to get query params.

Reply:
Why we need to use `url.parse` if params already parsed in the query variable?